.eslintrc.json

```diff
@@ -21,7 +22,7 @@
     "ecmaVersion": 11,
     "sourceType": "module"
   },
-  "plugins": ["react", "react-hooks", "prettier"],
+  "plugins": ["react", "react-hooks", "prettier", "cypress"],
```

Comment:
Add `prettier` as the last plugin. It was recommended in their docs to be last for formatting to work as expected

Reply:
Thanks for the tip, I changed it.